core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client}
+  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client.  This protocol falls back to Java Web Start Agent Protocol/2 (insecure) when it can't create a secure connection.  This protocol is not recommended.  Use Java Web Start Agent Protocol/4 instead.}
```

Comment:
I think `core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly` should reference `Java Web Start Agent Protocol/2 (insecure)` and `Java Web Start Agent Protocol/4` from their Display Names in `core/src/main/resources/jenkins/slaves/Messages.properties`.

That would improve the localization.  I don't know how to do this since I'm new to Jelly.  Any hints would be great.